lib/pbench/server/database/database.py

```diff
@@ -61,7 +65,18 @@ def init_db(server_config: PbenchServerConfig, logger: Logger):
         # nothing.
 
         engine = create_engine(db_uri)
-        Database.Base.metadata.create_all(bind=engine)
+        db_url = urlparse(db_uri)
+        if db_url.scheme == "sqlite":
+            Database.Base.metadata.create_all(bind=engine)
+        else:
```

Comment:
Does that mean `alembic` can't handle setting up the `sqlite3` schema in unit test mode? That's a shame, because I hate having test-specific conditionals in the real code.

Reply:
I didn't want to make the unit tests even longer such that each test would run an alembic migration.  Seemed like a bit of overkill.

Reply:
The downside being that our real code now says "we support sqlite3" but if someone tried to actually deploy with it, we'd be skipping the DB upgrade step.

Reply:
Okay, I'll propose a fix for that.  I think I have an idea.